src/components/TodoList/TodoList.tsx

```diff
@@ -1,100 +1,98 @@
-import React from 'react';
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
 
-export const TodoList: React.FC = () => (
-  <table className="table is-narrow is-fullwidth">
-    <thead>
-      <tr>
-        <th>#</th>
-        <th>
-          <span className="icon">
-            <i className="fas fa-check" />
-          </span>
-        </th>
-        <th>Title</th>
-        <th> </th>
-      </tr>
-    </thead>
+interface Props {
+  todos: Promise<Todo[]>,
+  isLoaded: boolean,
+  setIsLoaded: React.Dispatch<React.SetStateAction<boolean>>,
+  setModalIsOpened: React.Dispatch<React.SetStateAction<boolean>>,
+  setCurrentUserId: React.Dispatch<React.SetStateAction<number>>,
+  setCurrentTodo: React.Dispatch<React.SetStateAction<Todo>>,
+  filterCallback: (todo: Todo) => boolean,
+}
 
-    <tbody>
-      <tr data-cy="todo" className="">
-        <td className="is-vcentered">1</td>
-        <td className="is-vcentered" />
-        <td className="is-vcentered is-expanded">
-          <p className="has-text-danger">delectus aut autem</p>
-        </td>
-        <td className="has-text-right is-vcentered">
-          <button data-cy="selectButton" className="button" type="button">
-            <span className="icon">
-              <i className="far fa-eye" />
-            </span>
-          </button>
-        </td>
-      </tr>
-      <tr data-cy="todo" className="has-background-info-light">
-        <td className="is-vcentered">2</td>
-        <td className="is-vcentered" />
-        <td className="is-vcentered is-expanded">
-          <p className="has-text-danger">quis ut nam facilis et officia qui</p>
-        </td>
-        <td className="has-text-right is-vcentered">
-          <button data-cy="selectButton" className="button" type="button">
-            <span className="icon">
-              <i className="far fa-eye-slash" />
-            </span>
-          </button>
-        </td>
-      </tr>
+export const TodoList: React.FC<Props> = ({
+  todos,
+  isLoaded,
+  setIsLoaded,
+  setModalIsOpened,
+  setCurrentUserId,
+  setCurrentTodo,
+  filterCallback,
+}) => {
+  const [visibleTodos, setVisibleTodos] = useState<Todo[]>([]);
```

Comment:
let's just use `todos` from props and don't create additional state variable. you can use `useMemo` hook to filter `visibleTodos`